lib/rubocop/cop/layout/block_end_newline.rb

```diff
@@ -60,7 +60,8 @@ def message(node)
         end
 
         def last_heredoc_argument(node)
-          return unless (arguments = node&.arguments)
+          return unless node&.send_type?
```

Comment:
```suggestion
          return unless node&.call_type?
```

`call_type` includes &. `csend` nodes too.